server/sonar-server/src/test/java/org/sonar/server/ui/ws/OrganizationActionTest.java

```diff
@@ -204,6 +206,24 @@ public void returns_provisioning_true_when_user_can_provision_projects_in_organi
     verifyResponse(executeRequest(org2), false, true, false);
   }
 
+  @Test
+  public void returns_project_visibility_private() {
+    OrganizationDto organization = dbTester.organizations().insert();
+    dbClient.organizationDao().setNewProjectPrivate(dbTester.getSession(), organization, true);
+    dbTester.commit();
+    userSession.logIn().addPermission(PROVISION_PROJECTS, organization);
+    assertJson(executeRequest(organization).getInput()).isSimilarTo("{\"organization\": {\"projectVisibility\": \"private\"}}");
+  }
+
+  @Test
+  public void returns_project_visibility_public() {
+    OrganizationDto organization = dbTester.organizations().insert();
+    dbClient.organizationDao().setNewProjectPrivate(dbTester.getSession(), organization, false);
```

Comment:
This has been done in a later commit, but as discussed I should search for all occurrences and replace them all with the new method.